src/main/resources/assets/pedestalcrafting/lang/fr_fr.lang

```diff
@@ -0,0 +1,44 @@
+# GUI
+itemGroup.pedestalcrafting=Pedestal Crafting
+
+config.displaySpeed=Affichage de la vitesse
+config.displaySpeed.tooltip=Vitesse à laquel les éléments tournent, s'élèvent et tombent sur les piédestal - rotations/sec
```

Comment:
"laquel" devrait être "laquelle"

Reply:
You're right, I corrected it in my last commit

Reply:
You're welcome, sorry for the mistakes.